examples/docker-rootful.yaml

```diff
@@ -6,7 +6,7 @@
 # $ export DOCKER_HOST=$(limactl list docker-rootful --format 'unix://{{.Dir}}/sock/docker.sock')
 # $ docker ...
 
-# This template requires Lima v0.8.0 or later
+# This template requires Lima v0.20.0 or later
```

Comment:
This should happen after releasing v0.20, as some people may use the YAMLs in the master branch without installing Lima from the master branch.

Reply:
Split off template changes into #2123